mpu6050.c

```diff
@@ -0,0 +1,129 @@
+#include "mpu6050.h"
+#include <stdio.h>
+
+#include "hardware/i2c.h"
+#include "pico/stdlib.h"
+
+#define MPU6050_I2C_ADDR   0x68
```

Comment:
```suggestion
#define MPU6050_I2C_DEFAULT_ADDRESS   0x68
```
I believe you can change the I2C address of the mpu6050.